style-guide/tertiary.html

```diff
@@ -131,14 +131,14 @@
 										<li id="menu-item-129"><a href="clinical-partners/partner-with-us/">Partner With Us</a></li>
 									</ul>
 								</li>
-								<li id="menu-item-120"><a href="about/">About</a>
+								<li id="menu-item-513" class="non-anchor parent"><span class="parent-anchor">About</span>
 									<ul class="sub-menu">
-										<li id="menu-item-126"><a href="about/mission-meeting-states-needs/">Mission / Meeting State’s Needs</a></li>
-										<li id="menu-item-123"><a href="about/facts-figures/">Facts &#038; Figures</a></li>
-										<li id="menu-item-122"><a href="about/diversity-inclusion/">Diversity &#038; Inclusion</a></li>
-										<li id="menu-item-125"><a href="about/making-of-a-medical-school/">Making of a Medical School</a></li>
-										<li id="menu-item-121"><a href="about/accreditation/">Accreditation</a></li>
-										<li id="menu-item-124"><a href="about/leadership-faculty/">Leadership &#038; Faculty</a></li>
+										<li id="menu-item-107"><a href="https://stage.medicine.wsu.edu/about/mission-meeting-states-needs/">Mission / Meeting State’s Needs</a></li>
```

Comment:
The use of a full domain here causes all of the menu items to have the external link icon, which doesn't necessarily reflect the production site.